### PR TITLE
Hg 846 Add build-dist npm command and vulcanize

### PR DESCRIPTION
**Pull request by kenkouot**
This PR introduces `npm run build-dist`, a convenience command that will create a concatenated version of all the components as well as build the static assets.

### CODE REVIEW

**Comment by lizlux** [comment]
https://github.com/Wikia/guidelines/blob/master/Process/PullRequests.md

At least provide a title! ;)  

But for real, I think PR titles and descriptions can be very helpful when returning to old PRs while troubleshooting future bugs.

File: dist/components/import.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<link rel="import" href="/vendor/polymer/polymer.html">
+<link rel="import" href="/components/top-bar/top-bar.html">
```
Comment (lizlux) [inline dist/components/import.html]: is this `/components` path correct? There's no `components` dir in the root of the style guide repo or the root of `gh-pages` app. 

Wondering under which circumstances this file would be used.

**Comment by lizlux** [comment]
Also please be sure to create a separate PR for compiled files (or include them in this PR, either way).

File: dist/components/import.html (hunk 1 above)
Comment (kenkouot) [inline dist/components/import.html]: Actually, this file is not meant to be used in this way. It's solely for creating the compiled import-built file. I should be clear about that in a comment.

Comment (lizlux) [inline dist/components/import.html]: wondering if it should just be deleted after the import-build file is created then.

**Pull request merged**